### DISCUSSION TITLE
iOS Frontend Developer Needed for BookPlayer prototype

### DISCUSSION THREAD

**Discussion by moeadham** [TortugaPower/BookPlayer#1031] [General]
Hi BookPlayer Community,

I'm working on a prototype/hobby project diverging from the usual audiobook approach, aiming for a more visually engaging experience. I've forked BookPlayer for this, and I'll be tackling the backend changes. I'm in need of a hand on the iOS frontend side of things.

This is paid contract work, with the cost flexible depending on experience. Time commitment is also flexible - can be full time, part time, weekends.

I'm well-acquainted with the GPLv3 license, and will of course upstream any relevant work. This project’s distinct approach makes contributing the entire thing to the mainline less feasible.

If you're familiar with the BookPlayer frontend codebase and interested in discussing further, feel free to reach out! 

email: m _@_ moeadham.com
twitter: @moeadham

Thanks

**Comment by GianniCarlo** [comment]
Hi @moeadham , if you have any suggestions for the app feel free to open an issue, but this is not the right place for a job listing, as it is meant to discuss the app and further improvement of it
